src/README.md

```diff
@@ -233,11 +226,11 @@ configuration file inside your repository with the
 option set to `true`. Make sure to specify your config via the `APP_CONFIG`
 environment variable.
 
-To change your source code in running container, use Docker's
-[exec](https://docs.docker.com/reference/commandline/exec/) command:
+To change your source code in running container, use podman's
+[exec](https://github.com/containers/libpod/blob/master/docs/podman-exec.1.md) command:
 
 ```
-docker exec -it <CONTAINER_ID> /bin/bash
+podman exec -it <CONTAINER_ID> /bin/bash
```

Comment:
Same here. Just a note that Docker command is actually the same.

Reply:
Agreed.